### DISCUSSION TITLE
Add 7.4.1 version of FreeSurfer

### DISCUSSION THREAD

**Discussion by JavierSanchez3** [neurodesk/neurodesk.github.io-hugo#365] [New Tool]
Hello everyone,

Is it possible to add the latest version (7.4.1) of FreeSurfer?

Best regards!

[https://surfer.nmr.mgh.harvard.edu/fswiki/rel7downloads](url)

**Comment by stebo85** [comment]
yes, container is building right now :)
https://github.com/NeuroDesk/neurocontainers/actions/runs/6241373288/job/16951546660

**Reply by JavierSanchez3** [reply]
Thanks for your soon response! Have a nice day! :)

**Reply by stebo85** [reply]
Dear @JavierSanchez3,

Freesurfer 7.4.1 is now available in Neurodesk. You can either update the neurodesktop container to the latest version (vnmd/neurodesktop:2023-09-20) or you can hit the update button in the menu system (depending on which version you are on right now, this might not work yet due to a bug). Alternatively you can access the new Freesurfer version through a terminal and run "ml freesurfer/7.4.1" - can you test and let me know if everything works as expected?

Thank you
Steffen

**Reply by JavierSanchez3** [reply]
Hi Steffen! I have used Freesurfer 7.4.1 among these days and, at the moment, everything goes well.

Thank you!

**Reply by stebo85** [reply]
Wonderful- thank you for the feedback!
